### PR TITLE
Add namespace resource quota

### PR DESCRIPTION
**Pull request by ramperher**
This targets this tnf test:

- access-control-namespace-resource-quota: namespaces under test require a resource quota.

Just applying the same we're using in tnf_test_example, for simplicity: https://github.com/redhat-cip/dci-openshift-app-agent/blob/master/samples/tnf_test_example/hooks/templates/resource_quota.yml.j2

### CODE REVIEW

**Comment by ramperher** [comment]
Test is passing, this is ready for review.

File: testpmd/hooks/templates/resource_quota.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+kind: List
+items:
+  - apiVersion: v1
+    kind: ResourceQuota
+    metadata:
+      name: pods-high
+      namespace: {{ cnf_namespace }}
+    spec:
+      hard:
+        cpu: "1000"
```
Comment (manurodriguez) [inline testpmd/hooks/templates/resource_quota.yml.j2:11]: maybe 1000 is a little bit too high, I was thinking maybe a 100 cores?

File: testpmd/hooks/templates/resource_quota.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        memory: 200Gi
+        pods: "10"
```
Comment (manurodriguez) [inline testpmd/hooks/templates/resource_quota.yml.j2:13]: I see the number of pods `10` is constant in all quotas, is that expected?

**Review by manurodriguez — APPROVED** [review]
Looks good to me, just minor questions

**Pull request merged**